forte/ontology_specs/metric.json

```diff
@@ -0,0 +1,39 @@
+{ 
+    "name": "metric", 
+    "definitions": [ 
+      {
+        "entry_name": "ft.onto.metric.BaseMetric", 
+        "parent_entry": "forte.data.ontology.top.Generics", 
+        "description": "A base metric entity, all metric entities should inherit from it.", 
+        "attributes": [ 
+          { 
+              "name": "metric_name", 
+              "type": "str" 
+          }
+        ]
+      },
+      { 
+        "entry_name": "ft.onto.metric.SingleMetric", 
```

Comment:
We will define some metrics as examples in this file later (such as accuracy), how about we add them now?

Reply:
I thought by assigning proper `metric_name` (e.g. `accuracy` or `loss_value`) metrics like accuracy or loss value could be covered by SingleMetric.

Reply:
I see. Having a `metric_name` is not a bad idea, maybe we should do that.

Reply:
The base `Metric` class already had this `metric_name`

Reply:
i see, now i understand